main_routers/live2d_router.py

```
@@ -966,3 +966,88 @@ def open_model_directory(model_name: str):
 
+@router.delete('/model/{model_name}')
+def delete_model(model_name: str):
+    """删除指定的Live2D模型"""
+    try:
+        # 查找模型目录
+        model_dir, _url_prefix = find_model_directory(model_name)
+        
+        if not model_dir or not os.path.exists(model_dir):
+            return JSONResponse(status_code=404, content={"success": False, "error": f"模型 {model_name} 不存在"})
+        
+        # 检查是否是用户导入的模型（只能删除用户导入的模型，不能删除内置模型）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)